app/Http/Controllers/SupportController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Router\TranslatedRoute;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Redirect;
+use Inertia\Inertia;
+use Inertia\Response;
+
+final class SupportController extends Controller
+{
+    public function index(Request $request): Response
+    {
+        $slackSuccess = $request->query->has('slackSuccess');
+        $slackError = $request->query->has('slackError');
+
+        return Inertia::render('Support/Index', [
+            'email' => Auth::user()?->email,
+            'slackSuccess' => $slackSuccess,
+            'slackError' => $slackError,
+        ]);
+    }
+    public function sendInvitation(Request $request): RedirectResponse
+    {
+        $botToken = config('slack.botToken');
+        $channelID = config('slack.channelID');
+        $email = Auth::user()?->email;
+
+        try {
+            $response = Http::withToken($botToken)
+                ->post('https://slack.com/api/conversations.inviteShared', [
+                    'channel' => $channelID,
+                    'emails' => $email,
```

Comment:
I now see in the slack docs that emails is supposed to be an array. So even if this works, it is better to follow the correct guidelines.

```suggestion
                    'emails' => [$email],
```